pypeit/tests/files/spec1d_b27-J1217p3905_KASTb_2015May20T045733.560.fits

```diff
@@ -1,4 +1,4 @@
                                                 TTYPE23 = 'BOX_COUNTS_SIG'                                                      TFORM23 = '1D      '                                                            TTYPE24 = 'BOX_COUNTS_NIVAR'                                                    TFORM24 = '1D      '                                                            TTYPE25 = 'BOX_MASK'                                                            TFORM25 = '1L      '                                                            TTYPE26 = 'BOX_COUNTS_SKY'                                                      TFORM26 = '1D      '                                                            TTYPE27 = 'BOX_COUNTS_RN'                                                       TFORM27 = '1D      '                                                            EXTNAME = 'SPAT0176-SLIT0175-DET01' / extension name                            END                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             @fOI�4{@O0�:��@����F��@�-`�X�@6W��?�^Ycu� @!�B�n�?y�[��]�@)K�4'
```

Comment:
Did you intend to commit this file?

Reply:
I fear we need it and the tests keep touching it.  Needs to be fixed.

Am raising an Issue.